oop_student_report_project.py

```diff
@@ -0,0 +1,91 @@
+'''
+Create a students report card class showing following detials as table
+- name
+- roll no.
+- Maths marks
+- physics marks
+- chemistry marks
+- eng marks
+- total percentage 
+
+Note: should not consider the class of the student in which
+ they are studying but the table will be showing the a group of students of one class. 
+
+You can add more methods such as ID function that shows the ID card of a Student A with RollNo. , Blood group, Address and Parents name, emailID.
+
+
+'''
+from prettytable import PrettyTable
+
+class Student:
+        
+    def __init__(self, name, r_num, m_marks, p_marks, c_marks, eng_marks, b_group, address, m_name, f_name, email):
+        self.name = name
+        self.r_num = r_num
```

Comment:
```suggestion
        self.roll_number = r_num
```
Try to give meaningful names to attributes and methods. The shorcut is too confusing.

Reply:
Also do it for all of them throughout the file.

Reply:
done